### ISSUE TITLE
USAgov banner is styled as a header

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3974]
## Area or Component
Likely application-wide, but seen on the patient experience

## Quick Summary
The banner at the top of the site ("An official website of the United States government. Here's how you know") is stylized as a header, but we normally have other headers in main content of the application.

## Accessibility Violation
Two:
* Ensures the document has at most one banner landmark (https://dequeuniversity.com/rules/axe/4.4/landmark-no-duplicate-banner?application=AxeChrome)
* Landmarks should have a unique role or role/label/title (i.e. accessible name) combination (https://dequeuniversity.com/rules/axe/4.4/landmark-unique?application=AxeChrome)

## Action Requested
If possible: remove the role=`banner` from the USA-gov banner at the top of the site.
Also change it away from a `<header>` element, since we nearly always have our own `<header>` element.

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/app/commonComponents/USAGovBanner.tsx#L80-L80

## Original Deque Issue:
landmark-no-duplicate-banner | Ensures the document has at most one banner landmark | Two header landmarks on the page; one for main content and one for the "an official website of the US govt" | Document should not have more than one banner landmark | https://dequeuniversity.com/rules/axe/4.4/landmark-no-duplicate-banner?application=AxeChrome | moderate | No | No |   | [".usa-banner__header"] | Fix any of the following:  Document has more than one banner landmark | <header class="usa-banner__header"> | cat.semantics,best-practice | rnv7@cdc.gov | Patient Self Reg - Terms of Service | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T20:53:04.535Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
landmark-unique | Landmarks should have a unique role or role/label/title (i.e. accessible name) combination | Same as above | Ensures landmarks are unique | https://dequeuniversity.com/rules/axe/4.4/landmark-unique?application=AxeChrome | moderate | No | No |   | [".usa-banner__header"] | Fix any of the following:  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable | <header class="usa-banner__header"> | cat.semantics,best-practice | rnv7@cdc.gov | Patient Self Reg - Terms of Service | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T20:53:04.544Z

**Comment by emmastephenson** [comment]
Can confirm; this is seen application-wide.
